style.css

```diff
@@ -0,0 +1,976 @@
+  margin-top: -25px;
+  margin-bottom: 30px;
+  margin-left: 45%;
+}
+
+.maka-d {
+  padding-left: 8px;
+  display: flex;
+  flex-direction: column;
+}
+
+.maka-d:nth-child(even) {
+  display: none;
+}
+
+.maka-box {
+  padding-left: 8px;
+  display: flex;
+  flex-direction: row;
+}
+
+.onye-okwu {
+  display: flex;
+  flex-direction: row;
+  margin: 0 10px;
+  padding: 10px 10px;
+}
+
+.onye-okwu img {
+  width: 150px;
+  height: 150px;
+  padding: 0 0;
+}
+
+.okwu {
+  margin-left: 5%;
+  margin-right: -10px;
+  padding: -30px 12px;
+  width: 100%;
+}
+
+.okwu h2 {
+  margin-bottom: 7px;
+  font-family: "Lato", sans-serif;
+  font-size: 22px;
+  font-weight: 700;
+}
+
+.okwu p {
+  width: 100%;
+  opacity: 1;
+}
+
+.head-us {
+  margin-bottom: 10px;
+}
+
+.first-speak {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 25px;
+  margin: 20px 0;
+  padding: 10px 0;
+}
+
+.btn {
+  width: 90%;
+  justify-content: center;
+  height: 50px;
+  color: #000;
+  background: #fff;
+  text-align: center;
+  margin: 40px 25px;
+  padding: 10px 10px;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  cursor: pointer;
+  border: 1px solid #d3d3d3;
+}
+
+.btn:hover {
+  background-color: #63666d;
+  color: #fff;
+  border-radius: 0;
+}
+
+.ukwu-cont {
+  background-color: #272a31;
+}
+
+.ukwu-head h2 {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 30px;
+  margin: 20px 0;
+  padding: 40px 0 10px 0;
+  color: #fff;
+}
+
+.ukwu-box {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 20px;
+  margin: 40px 0;
+  padding: 30px 0;
+}
+
+.part {
+  filter: invert(100%);
+  width: 100px;
+}
+
+.aka-cont {
+  padding: 30px 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.aka-box {
+  display: flex;
+  flex-direction: row;
+}
+
+.aka-box img {
+  width: 100px;
+  height: 100px;
+}
+
+.aka-box p {
+  padding-top: 25px;
+  margin-left: -20px;
+  font-weight: 700;
+}
+
+.aka-text p {
+  margin-top: 30px;
+}
+
+.about-text {
+  text-align: center;
+  margin-left: 0;
+  width: 100%;
+}
+
+.about2 {
+  text-align: center;
+  margin-top: 40px;
+  line-height: 30px;
+  width: 90%;
+  margin-left: 30px;
+}
+
+.ncheta {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.line2 {
+  width: 35px;
+  height: 2px;
+  align-self: baseline;
+  background-color: rgba(160, 159, 159, 1);
+  margin: 12px 0;
+}
+
+.akwukwo {
+  position: relative;
+  opacity: 0.8;
+}
+
+em {
+  color: #ec5242;
+}
+
+.h-img {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  display: block;
+  overflow: hidden;
+  z-index: -1;
+  height: 100%;
+}
+
+.intro {
+  padding: 80px 20px;
+  text-align: center;
+  color: #fff;
+}
+
+.intro h3 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #fff;
+}
+
+/* mobile view */
+
+@media screen and (max-width: 768px) {
+  .hamburger {
```

Comment:
- [ ] Kindly make sure The CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around as it is the best practice and helps to reduce the code lines.
The media queries is supposed to be:
```css
@media screen and (min-width: 768px)
```